packages/frontend/src/pages/common/icons/assets/UsdcIcon.tsx

```diff
@@ -0,0 +1,31 @@
QVMjiMyYa5dCHYiIJAJ/qZPMCq1dHjk38g3zBntCMXhTrskRIYEvZOothxqUAmXMCglXJmmkWAvcqLb8GwsY2KumTiSm4saiFohlCHOn0RxZxWluE76/f2B5kMmSIwhPZ6ljT8GTTFRMkHSXIMViRSOwgvHZ5CIRmnEmq74VLOZXCDRh9qALcO8QiLIvaHB40DIaOKvUv8z1GAlzM6AHw6Nlfko6Y2ygyHXjQ/3lYGwr080Selpl3QgcgUD4m4GTjJrAGmyFqaW4QB1MeaMw3KcWdaACiQ+EhNMeiRn+5EGuXNCiiI+ATVZ0fz+8pOJqXDQFmHmPnXBp1WGXhJo1GrBGOFz4jxTd1FAKbEhoMGn6R7ndIlxwO5DzZcRSZRfMmto8VgmMeMcGdxzyiXLN1XKg6iMGa5pXvn9ItTHIKoBvkbGVqFCpiP3ViXzDAVU3+Uj/E2aiMsd2Co4LesGmhA5N4CSuRR44BDvbj0nR5Esbx6fpCIYfSEEJQqBBCREOmTCkBgZjzDMnPOHtafFxaKwl8v9cVPAwfaSEequx1CBdLf72W84pBvsAoMGrhPppoP1sh1Ri9XrANh0/8Qw5W/DoBk6awRyCpljHVUCP/nwq/NBTZZ3E7cBHQgQQWTagbpWGkB3D0bYU7DuMSAZCSkgo3av+r4YNRQZYvlRVtOuIg51SZ+juVEbT8zJkn7xocJicgryFM0smMYZdIjbgDqYmeVeTRohai0gvTJWM1EZMAA5JZQsBwGW+YxU7N8n58VJGKROImhOzCM0VTr4OlTNa9E0GZAKrbF0CWa4LDKpVQNYLVgygqBJ0zqCzxgzjQx7ndAMzYCDBdo7IwgJ9AeqmmXMC7flKHNqbvfIQ+8fk60oT8iEAUbo9ndelOFBEyLvTQRO9iUEOrh/KO5JhRyGRvTdEJlvHtO1SjOi07LaJZBAOq9VZiepJK660beXkWXOqERl28rcQBA//B6D0Mx6y4AT1a2q4bLSNXGNRN1GMyRUo5s8ApoCTVzTkVt2Ozja94qnHrgSDAQUJGudvciiLL9pz7MK4mXjgTqlJX+gpOsRntfBZEZmRSmN3YievgXRiIUfhcETFMbt16QoEyQwwa5D/UZJW+UnVr+zRsZ3Rz221ZUWFgH89bpN3+nQIsMOoDk5BLFN8CewA/3lZ/vr1NiH5jURBLD/3oG+wIyYEl2BApClxPUY6TCH3ImehC0utUXpsq5imUHg3e9trRI2aea7uE7tjcuJlkldTFN4pKUd5wL363jXEl1qR/PemjTxPvYiUxSWLmDEPt/klvE5UWiWQhVxsTCNz1HIsbQwbJ7b+b0VvQVLD9o297PapSPzDLM9z5NADuReR/n+r+3Vls0VzZAjod9iXja8n68lYJh+c2Mlis3AZgxm9d6hpYVrlYhSJ93r8urafHz1peEu4lix4p9MGRarHJdzLFbCe6Bms10rZmBOrHw3H2xQE0eORNlgEYrLmpEAK2QZsv+OY02YgF4sKNUGP7gDpc4dE1KV9Cm019afUefNZQ1Q1toctmfPf/HWGV+2CYvcB0I+M1/Ig/SLjfvr1RJ9hT0Fm5yAchrrZIb+cAL5g3Mu+pgBJaeb1fyLY1H6kVkQ3EeNTcuPkwfQXhsVwf++qVNmZWaC+zFW39A/PXYvvytG+HYIb4reASU9BsRERq38FabACpgQjzLjqXnZMm04mfERrhb8gIQ6oOse5gfSKWH/Z8YRMs+7rkMbvHhefxSdviRIMyRuF0K6BeONms32zrqHh7Tznu4xR8qMh8+Iu3MgbHS0P1EMdxeaO+UsIgZ76AjYPocCHCwQAcM0I485LJI4Nk9xEXY5jY5xPaR6HISZGWFpzwjF9vdJTDHTukyROJ9fWS5fn2yyLN0hkv/kZMa9fmzdX9SArgejeY/vKtC9+grIeZA/M/xOJ5w9OkH15bR9Apsjzrj+saVKxXgSZY4qhomYwzbNiL3FEDRwHOQxZBsjJ76WeHFVuZrQW2iCqE86bI75B5aO2WfQeXHwhCc1296qTp70AGxI54yrR2VMSZLYctRQBahcY9E90meG4RXBVPTrHEI3tLqV9qgNA2gy/PgDTY8R6Z5rUzEMz5KrB0WruwgsQ14AEyxnrJnQ2kWzPX3w5bFr/e/swQgXRsxY9I2rozUZyhqPnz3u4o9jMI89cOJ+mF8x61KqNLUxyDFTwNBe9OqcnFh3cj6HHjMgWl6/ezCGG3HduYUa+hLRrAjTxm1HOFr1J/Pid9z7dnxa2ssVn/7Z18x3LQUwUrb5VXfGzF/t8nR6hmJPnvk6ImDk4WCAb5uY+Zkr+Jy5YwfC76HK3l70YESLZDd/3MU5MrvH5Zuq5JVPz6iK2WyeBg3AscbhCF9YvHiIL4z6EWfZWPGN0LCnih/BG6JwvikyM0NE7Br/vuWcbDhQr8qXWQUJkpkYiirXunbqJgZfWAnXo02oQABZX1on/8V7FY6QqIRgTGBxo9MZ+nDJ8wXnzPTwdxAF+raCmf56p+c1ODreI1q/DFX2jw3xKEEYccohXdZZvV3MgpKvGljP+foeTvOtyFMkYiytr/UxUXgy6K5gC8ZzvLNL4esuuOzP4cUBr7yMfYxqrKk05opLXKp4hwQYsoP5groCr6dRnr9rd4QV7V9irQkD1aVwqn18Z1fVWnuX7vUsws8c4/D3+RcMloHLl+JT097fsTDLYlbaE3ufGDGOMM8gWSyERG/Gs5jvgxEw0AQrXuEIsf1x/5KxJX3FcVmM8FK+9S07Wz+JL1nw+msGmOruY/qK1GofCKmFsW2EZS4fkBm3xSg9rPZaPbtsRoxLyFB5RX0h/l+lOXCN6TDoXDAVC0fxGyRZoex6xq5Ul0YQcBha2KDZtdUsAC+XAQPDFTNiXAAH10Y89028x+Uq0DAV50AZa7lw1HQQSvNDHc+Ix4wsjXhWCcKPwM13sbMTLar04JK8Wjg+Y8AVw/8B+Jid0B6zud4AAAAASUVORK5CYII="
```

Comment:
Not sure if I like inlined pngs that much. I think that having all asset icons as statically served optimized 64x64 pngs should be fine, it's just 2 or 3 kb per file.

Reply:
That's a mistake - not every icon was a "proper" svg in figma, I thought I updated all png-based ones, apparently I missed this one :) will find a proper svg for usdc online.